### PR TITLE
Workast #3839

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/workast/actions/create-task/create-task.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    dueDate: {
+      label: "Due Date",
+      description: "The task due date. `E.g. 2022-04-04T17:30Z`",
+      type: "string",
+      optional: true,
+    },
+    listPosition: {
+      label: "List Position",
+      description: "The task position within the list",
+      type: "string",
```
Comment (andrewjschuang) [inline components/workast/actions/create-task/create-task.ts:53]:
```suggestion
      type: "integer",
```

**Review by andrewjschuang — COMMENTED** [review]
@lcaresia thanks for the contribution!

I've added a comment, but moving to QA

**Comment by DilanAthukorala** [comment]
@lcaresia Create Space action has invalid field prop - List/Space Id
image

**Comment by lcaresia** [comment]
@DilanAthukorala Removed!

**Comment by DilanAthukorala** [comment]
Verification completed and this is ready for release.

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**